CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+## Contributing
+Thank you for your interest in contributing to Codewind.
+
+#### Before you contribute
+- Contributions must follow the Eclipse organization specifications. Refer to the [Git commit instructions](https://www.eclipse.org/projects/handbook/#resources-commit).
+
+#### Adding contributions
+- See the [Codewind GitHub Workflows](https://wiki.eclipse.org/Codewind_GitHub_Workflows) wiki.
+
+## Raising an issue
+
+#### Before you raise an issue
+- Search the [list of issues](https://github.com/eclipse/codewind-eclipse/issues) to see if your issue has already been raised.
+- See the [Codewind documentation](https://www.eclipse.org/codewind/docindex.html) to see if existing documentation includes the issue.
```

Comment:
Is the goal to point them to the troubleshooting page: https://www.eclipse.org/codewind/troubleshooting.html or you want to point to the main doc page?  The reason that I ask is the description say "includes the issue" sounds like the trouble shooting page to me.